src/builder/commands/pip.rs

```diff
@@ -326,11 +341,18 @@ fn version_req(hash: &mut Digest, fname: &Path) -> Result<(), VersionError> {
                 if chunk.len() == 0 || chunk.starts_with("#") {
                     continue;
                 }
+                if let Some(req) = parse_req_filename(chunk) {
+                    included_files.push(req);
+                }
                 // Should we also ignore the order?
                 hash.item(chunk);
             }
             Ok(())
-    }).map_err(err)
+    }).map_err(err));
+    for req in &included_files {
```

Comment:
I couldn't make a recursive call in loop because of:
```
   Compiling vagga v0.6.0 (file:///work)
<std macros>:6:1: 6:32 error: the trait bound `std::io::Error: std::convert::From<version::error::Error>` is not satisfied [E0277]
<std macros>:6 $ crate:: convert:: From:: from ( err ) ) } } )
               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/builder/commands/pip.rs:331:25: 332:67 note: in this expansion of try! (defined in <std macros>)
<std macros>:6:1: 6:32 help: run `rustc --explain E0277` to see a detailed explanation
<std macros>:6:1: 6:32 help: the following implementations were found:
<std macros>:6:1: 6:32 help:   <std::io::Error as std::convert::From<xz2::stream::Error>>
<std macros>:6:1: 6:32 help:   <std::io::Error as std::convert::From<nix::Errno>>
<std macros>:6:1: 6:32 help:   <std::io::Error as std::convert::From<nix::Error>>
<std macros>:6:1: 6:32 help:   <std::io::Error as std::convert::From<zip::result::ZipError>>
<std macros>:6:1: 6:32 help: and 4 others
<std macros>:6:1: 6:32 note: required by `std::convert::From::from`
error: aborting due to previous error
``` 
Is there a way to fix it differently? I don't think it should be commented.